modules/appeals_api/app/swagger/appeals_api/v2/swagger_dev.json

```diff
@@ -4514,26 +4515,6 @@
                           "email": "bob@bobbytablesemail.com",
                           "timezone": "America/Chicago"
                         },
-                        "claimant": {
```

Comment:
This is really bizarre. Notice of Disagreement v2 is _**not**_ currently displayed on dev-developer-portal.

_the reason this object was removed was **because** it was not displayed on dev-developer-portal_

[Seen here](https://dev-developer.va.gov/explore/appeals/docs/decision_reviews?version=current)

So, the question is, was NOD v2 non-Veteran claimant supposed to be displayed on dev-developer-portal but wasn't displaying for a reason unrelated to us (again)?

Reply:
@cilestin

Reply:
I think we need to consider removing rswag. This is getting ridiculous.